src/main/java/frc/robot/Constants.java

```diff
@@ -13,7 +20,106 @@
+
+  public static final double PLACEHOLDER_DOUBLE = 1.0;
+  public static final int PLACEHOLDER_INT = 1;
+
   public static class OperatorConstants {
-    public static final int kDriverControllerPort = 0;
+    public static final int DRIVER_CONTROLLER_PORT = 0;
+  }
+
+  public static final class ModuleConstants {
+    /**The MAXSwerve module can be configured with one of three pinion gears: 12T, 13T, or 14T.
+     * This changes the drive speed of the module (a pinion gear with more teeth will result in a
+     * robot that drives faster).*/
+    public static final int DRIVING_MOTOR_PINION_TEETH = 14;
+
+    /**Invert the turning encoder, since the output shaft rotates in the opposite direction of
+     * the steering motor in the MAXSwerve Module.*/
+    public static final boolean TURNING_ENCODER_INVERTED = true;
+
+    /**Calculations required for driving motor conversion factors and feed forward*/
+    public static final double DRIVING_MOTOR_FREE_SPEED_RPS = NeoMotorConstants.FREE_SPEED_RPM / 60;
+    public static final double WHEEL_DIAMETER_METERS = 0.0762;
+    public static final double WHEEL_CIRCUMFERENCE_METERS = WHEEL_DIAMETER_METERS * Math.PI;
+    /**45 teeth on the wheel's bevel gear, 22 teeth on the first-stage spur gear, 15 teeth on the bevel pinion*/
+    public static final double DRIVING_MOTOR_REDUCTION = (45.0 * 22) / (DRIVING_MOTOR_PINION_TEETH * 15);
+    public static final double DRIVE_WHEEL_FREE_SPEED_RPS = (DRIVING_MOTOR_FREE_SPEED_RPS * WHEEL_CIRCUMFERENCE_METERS)
+        / DRIVING_MOTOR_REDUCTION;
+
+    public static final double DRIVING_ENCODER_POSITION_FACTOR_METERS = (WHEEL_DIAMETER_METERS * Math.PI)
+        / DRIVING_MOTOR_REDUCTION; // meters
+    public static final double DRIVING_ENCODER_VELOCITY_FACTOR_METERS_PER_SECOND = ((WHEEL_DIAMETER_METERS * Math.PI)
+        / DRIVING_MOTOR_REDUCTION) / 60.0; // meters per second
+
+    public static final double TURNING_ENCODER_POSITION_FACTOR_RADIANS = (2 * Math.PI); // radians
+    public static final double TURNING_ENCODER_VELOCITY_FACTOR_RAD_PER_SEC = (2 * Math.PI) / 60.0; // radians per second
+
+    public static final double TURNING_ENCODER_POSITION_PID_MIN_INPUT_RADIANS = 0; // radians
+    public static final double TURNING_ENCODER_POSITION_PID_MAX_INPUT_RADIANS = TURNING_ENCODER_POSITION_FACTOR_RADIANS; // radians
+
+    public static final double DRIVING_P = 0.04;
+    public static final double DRIVING_I = 0;
+    public static final double DRIVING_D = 0;
+    public static final double DRIVING_FF = 1 / DRIVE_WHEEL_FREE_SPEED_RPS;
+    public static final double DRIVING_MIN_OUTPUT = -1;
+    public static final double DRIVING_MAX_OUTPUT = 1;
+
+    public static final double TURNING_P = 1;
+    public static final double TURNING_I = 0;
+    public static final double TURNING_D = 0;
+    public static final double TURNING_FF = 0;
+    public static final double TURNING_MIN_OUTPUT = -1;
+    public static final double TURNING_MAX_OUTPUT = 1;
```

Comment:
Let's reorganize as we did for the offseason robot. CAN IDs and controllers go in RobotMap, these would go in calibrations, etc.